components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/resources/dbScripts/h2.sql

```diff
@@ -747,6 +747,17 @@ CREATE TABLE IF NOT EXISTS  IDN_SAML2_ASSERTION_STORE (
   PRIMARY KEY (ID)
 );
 
+CREATE TABLE IDN_SAML2_ARTIFACT_STORE (
+  ID INT(11) NOT NULL AUTO_INCREMENT,
+  SOURCE_ID BLOB NOT NULL,
+  MESSAGE_HANDLER BLOB NOT NULL,
+  AUTHN_REQ_DTO BLOB NOT NULL,
```

Comment:
Should these fields be BLOB?

Reply:
Yeah. SOURCE_ID and MESSAGE_HANDLER are byte arrays. AUTHN_REQ_DTO is a serialized data object.  Do we have any other alternatives?

Reply:
Are they still contain binary data, or some binary representation of String IDs? Storing them as blobs will cause performance issues if we are going to retrieve data by those values.